integration_tests/support/commands.ts

```diff
@@ -1,7 +1,22 @@
-Cypress.Commands.add('signIn', (options = { failOnStatusCode: true }, nonce = undefined, initialRequestUrl = '/') => {
-  cy.request(initialRequestUrl)
-  return cy.task('getSignInUrl', nonce).then((url: string) => {
-    cy.visit(url, options)
-    return cy.task('verifyJwtAssertionForToken')
-  })
-})
+Cypress.Commands.add(
+  'signIn',
+  ({
+    options = { failOnStatusCode: true },
+    nonce = undefined,
+    initialRequestUrl = '/',
+    hideCookieBanner = true,
+  } = {}) => {
+    if (hideCookieBanner) {
+      cy.hideCookieBanner()
+    }
+    cy.request(initialRequestUrl)
+    return cy.task('getSignInUrl', nonce).then((url: string) => {
+      cy.visit(url, options)
+      return cy.task('verifyJwtAssertionForToken')
+    })
+  },
+)
+
+Cypress.Commands.add('hideCookieBanner', () =>
+  cy.setCookie('cookie_policy', encodeURIComponent(JSON.stringify({ acceptAnalytics: 'no' }))),
```

Comment:
Cookie banner at the top of all pages in the integration tests was getting annoying so I've disabled it by default by setting the `cookie_policy` cookie in advance..